app/scripts/controllers/mmi-controller.ts

```diff
@@ -775,7 +773,7 @@ export default class MMIController extends EventEmitter {
     const extensionId = this.extension.runtime.id;
 
     const networks = Object.values(
-      this.networkController.state.networkConfigurationsByChainId,
+      this.#networkControllerState.networkConfigurationsByChainId,
```

Comment:
It looks like this is not using the live version of NetworkController state either. Can we get a fresh copy of NetworkController here?